CheatSheets/docker-cheatsheet.md

```diff
@@ -0,0 +1,54 @@
+## Table of Contents
+
+- [Conda CheatSheet for Developers](#conda-cheatsheet-for-developers)
```

Comment:
```suggestion
- [Docker CheatSheet for Developers](#docker-cheatsheet-for-developers)
```

Reply:
Fixed the title issue and added it to the readme